src/components/main/note/final-view/NoteFinalView.js

```diff
@@ -54,6 +56,11 @@ const NoteFinalView = ({ note, id, userName, newPageX, newPageY }) => {
   // hence, when drag starts, we calculate the distance between current top left of div ('origin') and the point we grabbed it
   // when we update the position in onDragEnd, we adjust for this distance to generate the expected effect
   const onDragStart = (event) => {
+    event.dataTransfer.setData('text/plain', id);
+
+    /* eslint-disable no-param-reassign */
+    event.dataTransfer.dropEffect = 'move';
```

Comment:
How is that used for?? dataTransfer and text/plain?? Factor out text/plain, since it's used in another file.